### PR TITLE
Members Photo Animation #124

### PR DESCRIPTION
**Pull request by rahilkapoor**
This is how it looks => https://youtu.be/LU3ett-Ef_Y

### CODE REVIEW

**Review by Vividh25 — COMMENTED** [review]
Good work!

**Review by swarajpure — CHANGES_REQUESTED** [review]
Great job indeed! 🎉 🥳 

Suggesting some changes, please check! 🙂

File: css/main.css [hunk 1]
```diff
@@ -286,6 +286,27 @@ footer {
...
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  margin-top: 6rem;
+  min-height: 450px;
+}
+
+.member-img {
+  padding: 0.6em;
+  width: 120px;
```
Comment (harshith-venkatesh) [inline css/main.css:300]: NIT: could we go with rem here... as it is good practise

File: js/animate.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+const width =
+  window.innerWidth ||
+  document.documentElement.clientWidth ||
+  document.body.clientWidth;
+const height =
+  window.innerHeight ||
+  document.documentElement.clientHeight ||
+  document.body.clientHeight;
+const constant = width / (height * 2.3);
```
Comment (harshith-venkatesh) [inline js/animate.js:9]: NIT: constant can be confusing , could we look for more relatable word like if it is related to div, constantDiv or constantBox I am not sure, you can look into it

File: js/animate.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  const deg = n * (72 * (Math.PI / 180));
+  return (width / 5) * Math.cos(deg);
+};
+
+const fetchy = (n) => {
+  const deg = n * (72 * (Math.PI / 180));
+  return 160 * Math.sin(deg);
+};
+
+const arr = document.querySelectorAll('.member_animation');
```
Comment (harshith-venkatesh) [inline js/animate.js:25]: just a suggestion, can be ignored - could we look for more relatable word here?

Comment (rahilkapoor) [reply js/animate.js:25]: Changed arr => memberList
I guess this will do. What do you say?

**Review by swarajpure — APPROVED** [review]
Approving from my side
Let @harshith-venkatesh have a look once 🙂 

Great job in here! 🔥

File: css/main.css (hunk 1 above)
Comment (harshith-venkatesh) [reply css/main.css:298]: You can close this comment

File: js/animate.js (hunk 3 above)
Comment (harshith-venkatesh) [reply js/animate.js]: Yeah this is better

**Review by harshith-venkatesh — APPROVED** [review]

**Review by Vividh25 — APPROVED** [review]

**Pull request merged**